.github/workflows/ci_tests_dev.yaml

```diff
@@ -57,6 +58,8 @@ jobs:
       # Install GMT master branch
       - name: Install GMT from master
```

Comment:
```suggestion
      # Build and install latest GMT from Github
      - name: Install GMT latest
```

Reply:
I just changed the step name to:
```
- name: Install GMT ${{ matrix.gmt_git_ref }} branch
```

Reply:
Oh cool, and could you update MAINTENANCE.md too at https://github.com/GenericMappingTools/pygmt/blob/master/MAINTENANCE.md#github-actions.